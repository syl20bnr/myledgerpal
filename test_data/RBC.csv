"Type de compte","Numéro du compte","Date de l'opération","Numéro du chèque","Description 1","Description 2","CAD","USD"
Chèques,00335-1234567,4/28/2014,,"Paiement","PAIEMENT W3 - 4732 ELECTRONIC BO",-95.25,,
Chèques,00335-1234567,4/28/2014,,"Paiement","PAIEMENT W3 - 7374 HYDRO QUEBEC ",-336.59,,
Chèques,00335-1234567,5/1/2014,,"COSTCO ESSENCE","ACHAT PDI ---- 6765 ",-52.69,,
Chèques,00335-1234567,5/1/2014,,"COSTCO WHOLESAL","ACHAT PDI ---- 2154 ",-96.32,,
Chèques,00335-1234567,5/5/2014,,"VERSEMENT SUR HYP",,-756.38,,
Chèques,00335-1234567,5/9/2014,000013,"CHEQUE No 013",,-60.00,,
Chèques,00335-1234567,5/12/2014,,"ASSURANCE","DESJ. ASS. GEN. ",-143.42,,
Chèques,00335-1234567,5/23/2014,,"FRAIS MENSUELS",,-10.95,,
Chèques,00335-1234567,7/14/2014,,"CREDIT INTERNE",,46.00,,