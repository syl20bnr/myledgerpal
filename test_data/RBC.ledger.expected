; -*- ledger -*-

2014/04/28 * Electronic Box
    Expenses:Unknown                                   $ 95.25
    Assets:Compte Joint RBC

2014/04/28 * Hydro Quebec
    Expenses:Unknown                                  $ 336.59
    Assets:Compte Joint RBC

2014/05/01 * Essence (Costco)
    Expenses:Transports:Voiture:Essence                $ 52.69
    Assets:Compte Joint RBC

2014/05/01 * Costco
    Expenses:Alimentation:Courses                      $ 96.32
    Assets:Compte Joint RBC

2014/05/05 * Hypotheque
    Expenses:Unknown                                  $ 756.38
    Assets:Compte Joint RBC

2014/05/09 * CHEQUE No 013
    Expenses:Unknown                                   $ 60.00
    Assets:Compte Joint RBC

2014/05/10 * Essence (Costco)
    Expenses:Transports:Voiture:Essence                $ 60.32
    Assets:Compte Joint RBC

2014/05/12 * Assurances Desjardins
    Expenses:Logement:Assurance                        $ 57.37
    Expenses:Transports:Voiture:Assurance              $ 86.05
    Assets:Compte Joint RBC                          $ -143.42

2014/05/23 * Frais Bancaires
    Expenses:Unknown                                   $ 10.95
    Assets:Compte Joint RBC

2014/07/14 * Remboursement Banque
    Assets:Compte Joint RBC                            $ 46.00
    Expenses:Unknown
